recipes/_configure_inputs.rb

```diff
@@ -11,15 +11,14 @@
 input_stanzas = CernerSplunk::LWRP.convert_monitors node['splunk']['monitors'], node['splunk']['main_project_index'], base_hash
 
 if %i[server cluster_slave].include? node['splunk']['node_type']
-  bag = CernerSplunk.my_cluster_data(node)
+  cluster, bag = CernerSplunk.my_cluster(node)
```

Comment:
Is it me, or is the introduced cluster variable not used?

Reply:
It is on line 23

Reply:
Aargh... who wrote this crap... oh yeah :bear: